server/tournaments/index.ts

```diff
@@ -1699,6 +1727,33 @@ const chatCommands: ChatCommands = {
 					}
 				}
 			}
+		} else if (cmd === 'export') {
+			if (room.lastTournamentData && room.lastTournamentData.type === 'tree') {
+				let tourString = '';
+				let tourData = [];
+				const tourWinner = room.lastTournamentData.rootNode.team;
+				let roundCounter = Math.ceil(Math.log(room.lastTournamentData.playersLength) / Math.log(2));
```

Comment:
(afaik) this doesn't work for multi-elim tours

Reply:
wait, do we support multielim tours?

Reply:
Yes.